nifi-toolkit/nifi-toolkit-tls/src/test/java/org/apache/nifi/toolkit/tls/util/TlsHelperTest.java

```diff
@@ -448,4 +460,58 @@ public void testClientDnFilenameSpecialChars() throws Exception {
         assertEquals("CN=testuser_OU=NiFi_Organisation", escapedClientDn);
     }
 
+    private KeyStore setupKeystore() throws Exception {
+
+        KeyStore ks = KeyStore.getInstance("JKS");
+        InputStream readStream = getClass().getClassLoader().getResourceAsStream("keystore.jks");
+        ks.load(readStream, "changeit".toCharArray());
+
+        return ks;
+    }
+
+    @Test
+    public void testOutputCertsAsPem() throws Exception {
```

Comment:
I'd prefer if these tests were more descriptive -- list the expectations for what is happening here. The keystore should have a known number of entries, and that number should match the number of outputs. In addition, the key signature should be compared with that queried from the keystore directly, not just asserted that it is non-null. The same goes for the key assertions in the subsequent test.